consensus/consensus_v2.go

```diff
@@ -33,8 +33,8 @@ func (consensus *Consensus) handleMessageUpdate(payload []byte) {
 		return
 	}
 
-	// stop receiving normal message when is in viewchanging
-	if consensus.mode.Mode() == ViewChanging && msg.Type != msg_pb.MessageType_VIEWCHANGE && msg.Type != msg_pb.MessageType_NEWVIEW {
+	// stop receiving message send to leader when is in viewchanging but accept other normal messages to avoid view change trap
```

Comment:
“stop receiving message send to leader”？ need to reword so it's more understandable.

Reply:
why only ignore PREPARE and COMMIT message in viewchange mode now?

Reply:
Since PREPARE and COMMIT are only the need receive, the validators will save the prepared and committed message in their log

Reply:
so should be “stop receiving message sent to leader”